wagn/lib/wagn/railtie.rb

```diff
@@ -0,0 +1,12 @@
+
+require 'decko/engine'
```

Comment:
I'm not sure how those things relate.  I'm willing to go with another theory of what should require what.  I think this file becomes a bit more important with the latest fixes on the generator branch.